tests/plugins/clipboard/manual/draganddropimageintable.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.17.2, bug, 4952
```

Comment:
Why don't we o put those tests also in `tableselection` plugin? The source issue have tests in placed there 🤔

Reply:
if we disable the`clipboard` plugin, the problem with DnD images in the table disappears, that's why I put this in the clipboard manual tests. I'll move it :)